internal/store/pod.go

```diff
@@ -1317,6 +1319,58 @@ func createPodStatusPhaseFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodStatusContainerReadyTimeFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGenerator(
```

Comment:
Please use `generator.NewFamilyGeneratorWithStability` and introduce as basemetrics.Alpha